evap/rewards/tests/test_tools.py

```diff
@@ -38,19 +38,37 @@ def test_semester_not_activated(self):
     def test_everything_works(self):
         SemesterActivation.objects.create(semester=self.course.semester, is_active=True)
         self.form.submit()
-        self.assertEqual(reward_points_of_user(self.student), settings.REWARD_POINTS_PER_SEMESTER)
+        self.assertEqual(reward_points_of_user(self.student), target_points(1.0))
 
     def test_semester_activated_not_all_courses(self):
         SemesterActivation.objects.create(semester=self.course.semester, is_active=True)
         mommy.make(Course, semester=self.course.semester, participants=[self.student])
         self.form.submit()
-        self.assertEqual(0, reward_points_of_user(self.student))
+        self.assertEqual(target_points(0.5), reward_points_of_user(self.student))
 
-    def test_already_got_points(self):
+    def test_already_got_grant_objects(self):
         SemesterActivation.objects.create(semester=self.course.semester, is_active=True)
         mommy.make(RewardPointGranting, user_profile=self.student, value=0, semester=self.course.semester)
         self.form.submit()
-        self.assertEqual(0, reward_points_of_user(self.student))
+        self.assertEqual(target_points(1.0), reward_points_of_user(self.student))
+
+    def test_already_got_points(self):
+        SemesterActivation.objects.create(semester=self.course.semester, is_active=True)
+        mommy.make(RewardPointGranting, user_profile=self.student, value=target_points(1.0), semester=self.course.semester)
```

Comment:
`already_got_grant_object` is the only test testing multiple grant objects per semester. This is a regression test, because this would not work before this PR.

`already_got_enough_points` is the only test testing that we dont grant points if the user has enough already.

I think both are important, but can't be combined.

Reply:
okay, maybe call the first test `test_already_got_grant_object_but_points_missing` and the second `test_already_got_grant_object_and_enough_points` or something like that?
maybe add assertions that there is (or there is not, respectively) two RewardPointGranting objects for that user and semester.